processing/src/main/java/io/druid/segment/data/GenericIndexed.java

```diff
@@ -496,11 +497,11 @@ public int indexOf(T value)
           allowReverseLookup = false;
         }
 
-        // for compatibility with the format, but this field is unused
```

Comment:
I don't like this design. It will make harder to remove this (in fact unused) field in the future. Instead, I suggest to encode nullness somewhere by applying bit inverse (~) to some field and writing it so.

Reply:
agreed, this field was unused but that does not mean we cannot use it to store nullability now when we need to do that. I don't see a need to remove this field in future since it is now being used.